classes/class-bsfrt-readtime.php

```diff
@@ -779,8 +779,17 @@ public function hook_header_top() {
 	 * @since 1.0.0
 	 * @return shortcode display value.
 	 */
-	public function read_meter_shortcode() {
+	public function read_meter_shortcode( $atts ) {
+		$atts = shortcode_atts( array(
+			'id' => ''
+		), $atts, 'read_meter' );
+		
 		$bsf_rt_post = get_the_ID();
+		if ( isset( $atts['id'] ) && ! empty( $atts['id'] ) && is_numeric( $atts['id'] ) ) {
+			if ( $post = get_post( sanitize_text_field( $atts['id'] ) ) ) {
```

Comment:
This might give error in phpcs, maybe change to this
```suggestion
            $post = get_post( sanitize_text_field( $atts['id'] ) );
			if ( $post ) {
```

Reply:
I tried but it did not give any error, might be due to improper standards setup. Changed in the latest commit.